config/nrfconnect/overlay-usb_support.conf

```diff
@@ -0,0 +1,25 @@
+#
+#   Copyright (c) 2020 Project CHIP Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+CONFIG_GPIO=y
```

Comment:
CONFIG_GPIO is already set I guess. Btw, do you think that we will need the overlay in other applications, not just pigweed-app?

Reply:
Yes I know, but honestly if it comes to overlays I like making them general. USB transport support it's quite general thing and we can imagine that we will need to use it again in some other case, so then we will just add this overlay and that's all. That's why I added CONFIG_GPIO, to make it independent from project settings. We may discuss our overlay policy later on, as I guess that everything will change anyway after moving CHIP to the NCS, where some useful overlays are currently defined.